apps/vscode/src/host/index.ts

```diff
@@ -73,14 +91,20 @@ function defaultExtensionHost() : ExtensionHost {
   return {
     executableLanguages: (visualMode: boolean, document: TextDocument, engine: MarkdownEngine) => {
       
-       const languages = executableLanguages();
-       const knitr = isKnitrDocument(document, engine);
+      const languages = executableLanguages();
+      const knitr = isKnitrDocument(document, engine);
 
-       // jupyter python (as distinct from knitr python) doesn't work in visual mode b/c
-       // jupyter.execSelectionInteractive  wants a text editor to be active
-       return languages.filter(language => knitr || !visualMode || (language !== "python"));
+      // jupyter python (as distinct from knitr python) doesn't work in visual mode b/c
+      // jupyter.execSelectionInteractive  wants a text editor to be active
+      return languages.filter(language => knitr || !visualMode || (language !== "python"));
     },
     cellExecutorForLanguage,
+    // in the default extension host, this is a noop:
+    registerStatementRangeProvider: (
+      engine: MarkdownEngine,
+		): vscode.Disposable => {
```

Comment:
Formatting seems a little strange here

Reply:
It would be good to get some automatic formatting on this repo @cscheid, like maybe add this?

```json
"[typescript]": {
		"editor.defaultFormatter": "vscode.typescript-language-features",
		"editor.formatOnSave": true
	}
```

Reply:
Yes. It'll cause a bit of source control churn to do it this late in the repo, but better late than never.

Reply:
(It's fixed on `main` now.)